Pods/CatchingFire/src/CatchingFire.swift

```diff
@@ -8,6 +8,10 @@
 
 import XCTest
 
+#if !swift(>=3.0)
```

Comment:
I used #if !swift(>=3.0) because that's what seems to be common in other reps. I think the rational is that when you only have some typealias to do, you want to clean the bulk of the code to comply to the latest version, so that it's cheaper to clean when you want to drop the older version.

continuing to use that style for the other cases was a matter of consistency.

if you still prefer the #if swift(<3.0), I think keeping the other way for typealias is best. what do you think?